obspy/io/rg16/__init__.py

```diff
@@ -1,20 +1,88 @@
 """
-Functions to read waveform data from Receiver Gather 1.6-1 foramt.
-This format is used for continuous data by Farfield's (fairfieldnodal.com)
-Zland product line (http://fairfieldnodal.com/equipment/zland).
+obspy.io.rg16 - receiver gather v1.6 read support for ObsPy
+=============================================================
 
-Inspired by a similar project by Thomas Lecocq
-found here: https://github.com/iceseismic/Fairfield-Receiver-Gather.
+Functions to read waveform data from Receiver Gather 1.6-1 format.
+This format is used for continuous data by
+`Farfield Nodal <fairfieldnodal.com>`_
+`Zland product line <http://fairfieldnodal.com/equipment/zland>`_.
 
-Some useful diagrams, provided by Faifield technical support, for
-understanding Base Scan intervals, data format, and sensor type numbers
-can be found here: https://imgur.com/a/4aneG.
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
 
-Note: Because there is not a standard deployment orientation, channel codes
-returned are 2, 3, and 4. See the link above to map these codes to
-the appropriate direction based on instrument orientation.
-"""
+Inspired by a similar project by a `similar script
+<https://github.com/iceseismic/Fairfield-Receiver-Gather`_.
+
+
+.. note::
+    1. Some useful diagrams, provided by Faifield technical support, for
+    understanding Base Scan intervals, data format, and sensor type numbers
+    can be found `here <https://imgur.com/a/4aneG>`_.
+
+    2. Because there is not a standard deployment orientation, channel codes
+    contained in the file format are 2, 3, and 4. See the above link for the
+    orientation in respect to instrument orientation. By default these are
+    mapped to Z, 1, 2 but a custom mapping can be passed to
+    :func:`obspy.io.rg16.read_rg16`.
+
+    3. The default polarity of channel 2 (vertical) is down. This is opposite
+    of what one might expect.
+
+Reading
+-------
+The rg16 format can be read using two methods:
+
+1. Using the standard :func:`~obspy.core.stream.read` function. Optionally,
+   the format parameter can be specified as "rg16" in order to avoid spending
+   time checking all the other foramts.
+
+2. Using the :mod:`obspy.io.rg16` specific function
+   :func:`obspy.io.rg16.read_rg16` function.
+
+There are two noteworthy parameters to the read_rg16 function:
```

Comment:
.. and also when using `read()`. 

You show it in the examples alright but it might not hurt to mention it also here right away.